src/pages/Commits.tsx

```diff
@@ -192,6 +203,12 @@ export const Commits = () => {
           hasFilters={hasFilters}
         />
       </PageContainer>
+      {!hasUsedMainlineCommitsBefore && (
+        <WelcomeModal
+          param="hasUsedMainlineCommitsBefore"
+          carouselCards={newMainlineCommitsUser}
+        />
+      )}
```

Comment:
I'm guessing that `hasUsedMainlineCommitsBefore` starts as `undefined` for a split second, so even if I have already seen this Modal, I get a flicker of it when visiting the page

https://user-images.githubusercontent.com/47064971/167450497-08e8615e-9082-4d03-94e9-2339a04ff1d5.mov

Reply:
Great catch!